content/educators/yung-qi.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Yung-Qi
+pronouns_en: he/him
+pronouns_nl: hij/hem
+quote:
+  Ik probeer niemand te overtuigen om anderen te accepteren die zich zelf zijn, maar ik probeer wel je horizon te verbreden om anderen lief te hebben zo als die zijn.
```

Comment:
```suggestion
  Ik probeer niemand te overtuigen om anderen te accepteren die zich zelf zijn, maar ik probeer wel je horizon te verbreden om anderen lief te hebben zoals die zijn.
```

Reply:
Ik ga even aan Yung vragen of dit prima is, het is immers een quote ;)